sstables/mx/writer.cc

```diff
@@ -1381,6 +1385,8 @@ void writer::write_clustered(const rt_marker& marker, uint64_t prev_row_size) {
     write_vint(*_data_writer, _tmp_bufs.size());
     flush_tmp_bufs();
     _collector.update_min_max_components(marker.position());
+
+    collect_range_tombstone_stats();
```

Comment:
This code-path is called from both `consume(clustering_row)` and `consume(range_tombstone_change)`. Need to detect which one it is and only bump the respective conunters.

Reply:
That is not the case because `consume(clustering_row)` calls `write_clustered(clustering_row)` and `consume(range_tombstone_change)` calls `write_clustered(rt_marker)`, so there is no overlap.